ert3/config/_stages_config.py

```diff
@@ -1,9 +1,32 @@
 import importlib
 import os
+import sys
 from typing import List, Callable, Optional
+
+try:
+    # Will only work from Python 3.8
+    from typing import Literal
+except ImportError:
+    from typing_extensions import Literal
+
 from pydantic import root_validator, validator, FilePath, BaseModel
 
 
+def _import_from(path):
+    if ":" not in path:
+        raise ValueError("Function should be defined as module:function")
+    module, func = path.split(":")
+    try:
+        module = importlib.import_module(module)
+    except ModuleNotFoundError as err:
+        raise ValueError(f"No module named: {module}") from err
```

Comment:
Why "degrade" this exception to `ValueError`? Why catch it at all? Isn't the `ModuleNotFound` _exactly_ right?